core/internal/httpserver/prometheus_test.go

```diff
@@ -151,6 +151,5 @@ func TestHttpServer_handlePrometheusMetrics(t *testing.T) {
 	assert.Contains(t, promExp, `burrow_kafka_topic_partition_offset{cluster="testcluster",partition="1",topic="testtopic"} 5566`)
 	assert.Contains(t, promExp, `burrow_kafka_topic_partition_offset{cluster="testcluster",partition="0",topic="testtopic1"} 54`)
```

Comment:
wasn't the idea to get a lag report for the topic `incomplete`? I think we are missing and assertion above as  `consumerPartitionLagGauge.With(labels).Set(float64(partition.CurrentLag))` will be called now even if `Complete` is below 1